src/app/hearing-part/components/listing-create/listing-create.component.ts

```diff
@@ -221,16 +266,18 @@ export class ListingCreateComponent implements OnInit {
         });
     }
 
-    afterClosed(confirmed) {
-        if (confirmed) {
-            this.createNotes();
-        }
-        if (this.editMode) {
-            this.afterEdit();
+    private prepareListingTypeData() {
+        switch (this.chosenListingType) {
+            case ListingTypeTab.Multi:
+                this.listing.hearing.duration = moment.duration(this.asDaysPipe.transform(this.listing.hearing.duration), 'days');
+                break;
+            case ListingTypeTab.Single:
+            default:
+                if (this.listing.hearing.duration && this.listing.hearing.duration.asMinutes() >= 24 * 60) {
```

Comment:
You could probably use safe function here instead of these 2 checks,
safe(() => this.listing.hearing.duration.asMinutes())